### PR TITLE
poke: 1.4 -> 2.0

### PR DESCRIPTION
**Pull request by trofi**
While at it added trivial updater.

### CODE REVIEW

**Comment by bobby285271** [comment]
@ofborg eval

**Comment by teto** [comment]
how do I test the updateScript (sry, never used) ?

File: pkgs/applications/editors/poke/default.nix [hunk 1]
```diff
@@ -65,6 +68,20 @@ in stdenv.mkDerivation rec {
...
 
+  passthru = {
+    updateScript = writeScript "update-poke" ''
+      #!/usr/bin/env nix-shell
+      #!nix-shell -i bash -p curl pcre common-updater-scripts
+
+      set -eu -o pipefail
+
+      # Expect the text in format of '<a href="...">poke 2.0</a>'
+      new_version="$(curl -s http://www.jemarch.net/poke |
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/poke/default.nix:79]:
```suggestion
      new_version="$(curl -s https://www.jemarch.net/poke |
```

**Comment by bobby285271** [comment]
> how do I test the updateScript (sry, never used) ?

Temporarily revert the update, then run `nix-shell maintainers/scripts/update.nix --argstr package poke` I think.

**Comment by trofi** [comment]
> > how do I test the updateScript (sry, never used) ?
> 
> Temporarily revert the update, then run `nix-shell maintainers/scripts/update.nix --argstr package poke` I think.

Yeah, that's exactly the command I used to update `poke`.

File: pkgs/applications/editors/poke/default.nix [hunk 2]
```diff
@@ -13,17 +13,20 @@
 , nbdSupport ? !stdenv.isDarwin, libnbd
 , textStylingSupport ? true
 , dejagnu
+
+# update script only
```
Comment (AndersonTorres) [inline pkgs/applications/editors/poke/default.nix:17]: The good old spoiler comment

**Pull request merged**